internal/otgutils/print.go

```diff
@@ -0,0 +1,72 @@
+package otgutils
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/ondatra"
+	otgtelemetry "github.com/openconfig/ondatra/telemetry/otg"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// PrintFlowMetrics is displaying the otg flow statistics.
+func PrintFlowMetrics(t *testing.T, otg *ondatra.OTG, c gosnappi.Config) {
+	var out strings.Builder
+	out.WriteString("\nFlow Metrics\n")
+	for i := 1; i <= 80; i++ {
+		out.WriteString("-")
+	}
+	out.WriteString("\n")
+	out.WriteString(fmt.Sprintf("%-25v%-15v%-15v%-15v%-15v\n", "Name", "Frames Tx", "Frames Rx", "FPS Tx", "FPS Rx"))
+	for _, f := range c.Flows().Items() {
+		flowMetrics := otg.Telemetry().Flow(f.Name()).Get(t)
+		rxPkts := flowMetrics.GetCounters().GetInPkts()
+		txPkts := flowMetrics.GetCounters().GetOutPkts()
+		rxRate := ygot.BinaryToFloat32(flowMetrics.GetInFrameRate())
+		txRate := ygot.BinaryToFloat32(flowMetrics.GetOutFrameRate())
+		out.WriteString(fmt.Sprintf("%-25v%-15v%-15v%-15v%-15v\n", f.Name(), txPkts, rxPkts, txRate, rxRate))
+	}
+	for i := 1; i <= 80; i++ {
+		out.WriteString("-")
+	}
+	out.WriteString("\n\n")
+	t.Log(out.String())
+}
+
+// PrintPortMetrics is displaying otg port stats.
+func PrintPortMetrics(t *testing.T, otg *ondatra.OTG, c gosnappi.Config) {
```

Comment:
Please make the same changes I recommended above.

Reply:
Changed both functions